### DISCUSSION TITLE
Can't seem to access sessions

### DISCUSSION THREAD

**Discussion by noamsiegel** [danielmiessler/Fabric#1100] [General]
Even though I have used fabric quite a bit, I don't have any sessions:

```
fabric --listsessions

No Sessions
```

Is anyone else having this issue? The documentation on sessions is lacking.

**Comment by bledburn** [comment]
By running the --sesion flag, if your sessions folder is empty or if the session name you use is not an existing session, then the fabric tool will create and start the session. Session management is sparse, so if you have knowledge of the jq command (in bash or zsh) you can take control of managing "inside" the .json session files. Otherwise, the fabric tool manages sessions by creating them on the fly, and then allowing you to list or wipe them. Again, to edit a session is something the tool does not do (that can find) and must be done manually (tools like jq are the easiest to use for this).
